scripts/codegen.sh

```diff
@@ -5,6 +5,7 @@ set -x
 
 poetry run python python/generate_schema.py > python/cli.schema.json
 yarn run json2ts python/cli.schema.json --no-additionalProperties > src/api/types.ts
+yarn prettier --config package.json --write src/api/types.ts
```

Comment:
Fixes weird pre-push loop between codegen and prettier: https://ref-studio.slack.com/archives/C05525DV9HN/p1687469412858809?thread_ts=1687468688.106519&cid=C05525DV9HN